content/INTRODUCTION.md

```diff
@@ -0,0 +1,25 @@
+**Requirements**: Knowledge in Basic PHP programming, Intermediate PHP programming, and Object-oriented Programming in PHP.
+
+**Duration**:
+
+**Standards**: K-12 Computer Science Framework Core Concepts and Practices
+* Algorithms and Programming
+    - Collaborating Around Computing
+    - Recognizing and Defining Computational Problems
+    - Developing and Using Abstractions
+    - Creating Computational Artifacts
+    - Testing and Refining Computational Artifacts
+
+## PHP Advance
+
+Extend and expand your PHP coding skills by learning how to apply the more advanced parts of the server-side scripting language.
+
+PHP advance is designed for:
+
+* Learners who have completed `An Introduction to Basic PHP Programming`, `Intermediate PHP Programming`, and `Object-Oriented Programming (OOP) in PHP`.
+
+* Individuals who have backgrounds in PHP programming.
+
+* Experienced programmers who want to take a refresher.
+
+In this course, you will gain an understanding of the essential topics for PHP advance such as ternary operator, type hinting, session, cookies, exceptions, and error handling. You will learn by answering questions and writing your own code. We skipped the lectures so that you will have more time to practice. Go beyond learning the syntax and experience how PHP advance works.
```

Comment:
Please separate this statement: Place this in the next line. Thanks.

> You will learn by answering questions and writing your own code. We skipped the lectures so that you will have more time to practice. Go beyond learning the syntax and experience how PHP advance works.